prov/efa/src/efa_ep.c

```diff
@@ -91,29 +91,37 @@ static int efa_ep_modify_qp_rst2rts(struct efa_qp *qp)
 				      IBV_QP_STATE | IBV_QP_SQ_PSN);
 }
 
-static int efa_ep_create_qp(struct efa_ep *ep,
-			    struct ibv_pd *ibv_pd,
-			    struct ibv_qp_init_attr *init_attr)
+static int efa_ep_create_qp_ex(struct efa_ep *ep,
+			       struct ibv_pd *ibv_pd,
+			       struct ibv_qp_init_attr_ex *init_attr_ex)
 {
 	struct efa_domain *domain = ep->domain;
```

Comment:
I would separate the declaration from the assignment.

Reply:
Good point. Fixed in new revision